src/integrations/admin/indexing-notification-integration.php

```diff
@@ -223,9 +226,13 @@ protected function should_show_notification() {
 		 * Show the notification when it is not in the hide notification period.
 		 * (E.g. when the user clicked on 'hide this notification for a week').
 		 */
-		$hide_until = (int) $this->options_helper->get( 'indexation_warning_hide_until' );
+		$hide_until = $this->options_helper->get( 'indexation_warning_hide_until', false );
+
+		if ( $hide_until === false ) {
+			return true;
+		}
 
-		return ( $hide_until !== 0 && $hide_until >= $this->date_helper->current_time() );
+		return ( $this->date_helper->current_time() > ( (int) $hide_until ) );
```

Comment:
This last line is not covered in the unit tests anymore.
Could you add a test that check that the notification is not shown when in the 'hide until' period?

Reply:
Added two tests for when the expiry time is exceeded and when it is not.